simulate/simulate.h

```diff
@@ -173,6 +181,7 @@ class Simulate {
   std::atomic_int uiloadrequest = 0;
 
   // loadrequest
+  //   3: display a loading message
```

Comment:
Why do we need "3" Isn't this already what "1" does (did 🙂) ?

Reply:
There are two stages to the loading process. Transitioning from state 3 to 2 is handled by the physics thread (this is optional for the user). Transitioning from 2 to 0 is handled by the UI thread.

The UI thread needs two states so it can put a loading message on the screen (transition from 2 to 1) prior to the blocking call to Simulate's LoadOnRenderThread (transition from 1 to 0).